helpers/middleman/menu.rb

```diff
@@ -264,6 +264,13 @@ def get_resources_menu(locale_obj: current_locale_obj)
                   label: 'Webinars & Events',
                   title: locale_page(page: 'events', locale_obj: locale_obj)[:page_title],
                   href: localize_url('events', locale_id: locale_obj[:id]),
+                },
+                {
+                  icon: 'users',
+                  icon_hover: 'users-fill',
+                  label: 'Community Forum',
+                  title: 'Community Forum',
+                  href: localize_url(base_url: config[:community_url], locale_id: locale_obj[:id]),
```

Comment:
Are you sure about this `localize_url`?  I think you might want a `unlocalized_url`.

This will send users to `test-community.sharesight.com/au/`.

Reply:
Oh right 🤦‍♂️